go/signedexchange/signedexchange.go

```diff
@@ -26,7 +26,41 @@ type Exchange struct {
 	payload []byte
 }
 
+// https://jyasskin.github.io/webpackage/implementation-draft/draft-yasskin-httpbis-origin-signed-exchanges-impl.html#stateful-headers.
+var (
+	statefulRequestHeaders = map[string]struct{}{
```

Comment:
Consider exporting these two sets. I'd like to offer packager users the option to remove these headers rather than error, as I imagine some web servers probably Set-Cookie automatically. (Let me know if you think that's a bad idea.)

Reply:
Good point. I'd add a new API to strip. I'd like to keep `NewExchange` as it is. The gen-signedexchange user would be able to choose to strip such headers by specifying flag. Does this make sense to you?

Reply:
Yep, that works for me.